superset-frontend/src/features/databases/DatabaseModal/index.tsx

```diff
@@ -1756,41 +1756,45 @@ const DatabaseModal: FunctionComponent<DatabaseModalProps> = ({
         onAddTableCatalog={() => {
           setDB({ type: ActionType.AddTableCatalogSheet });
         }}
-        onQueryChange={({ target }: { target: HTMLInputElement }) =>
+        onQueryChange={({ target }: { target: HTMLInputElement }) => {
           onChange(ActionType.QueryChange, {
             name: target.name,
             value: target.value,
-          })
-        }
-        onExtraInputChange={({ target }: { target: HTMLInputElement }) =>
+          });
+          handleClearValidationErrors();
```

Comment:
It seems that we are calling this in a bunch of places, can we just wrap the onChange in another handler and have that there in once?